lib-ltc/src/ltcdisplayrgb.cpp

```diff
@@ -42,6 +42,7 @@
 #include "ltcdisplayws28xxmatrix.h"
 //
 #include "ltcdisplayrgbpanel.h"
+#include "../../lib-ltc/include/h3/ltcoutputs.h"
```

Comment:
This cross-platform C++ file cannot be dependent of a H3 specific implementation.